packages/nimble-components/src/rich-text-editor/testing/rich-text-editor.pageobject.ts

```diff
@@ -156,10 +156,45 @@ export class RichTextEditorPageObject {
             .map(el => el.textContent || '');
     }
 
+    public isFooterHidden(): boolean {
+        const footerSection = this.getFooter()!;
+        return window.getComputedStyle(footerSection).display === 'none';
+    }
+
+    public getEditorTabIndex(): string {
+        return this.getTiptapEditor()?.getAttribute('tabindex') ?? '';
+    }
+
+    public async hideFooter(): Promise<void> {
+        this.richTextEditorElement.setAttribute('footer-hidden', '');
+        await waitForUpdatesAsync();
+    }
+
+    public async setDisabledState(): Promise<void> {
+        this.richTextEditorElement.setAttribute('disabled', '');
+        await waitForUpdatesAsync();
+    }
+
+    public isButtonDisabled(buttonIndex: ToolbarButton): boolean {
```

Comment:
Can you clean-up all the references to "index"? We aren't passing an index, we are passing a ToolbarButton.
So name the parameter `index` and can you clean-up the docs for all the other button helpers like `getButtonCheckedState`, etc to remove discussion of indexes?

Reply:
Did you mean to name the parament as `button: ToolbarButton` which is passed as an argument to these methods? As I could see in the other comment name it as a `button`: https://github.com/ni/nimble/pull/1424#discussion_r1300301659

I have removed the comments on discussing about the indexes in other helper methods.

Reply:
Sorry, yep, name the parameter `button` 👍